### DISCUSSION TITLE
Child Entities Lag Behind Parents

### DISCUSSION THREAD

**Discussion by occuros** [jcornaz/heron#230] [Q&A] [answered]
Using Heron `2.2.0`  with `2d feature` in bevy `0.6.1`:

When creating child entities (with sensors attached, and a visualization component like a circle from the lyon crate), the children lag behind the parent when using velocities to move the parent rigidbody which is of type `RigidBody::KinematicVelocityBased`.

When creating the same structure directly with rapier2d, the same lagging behind can't be observed, so I assume it has to do with heron directly. 

Is that a known issue, or is there something special that needs to be considered when using child entities with rigidbodies?

**Comment by jcornaz** [comment, marked as answer]
I think it is a problem in heron. I opened #231

**Reply by occuros** [reply]
Great to hear, heron has such a clean api surface, would be great to be able to use it.
